priv/perf/Makefile

```diff
@@ -2,14 +2,20 @@
 list:
 	@$(MAKE) -pRrq -f $(lastword $(MAKEFILE_LIST)) : 2>/dev/null | awk -v RS= -F: '/^# File/,/^# Finished Make data base/ {if ($$1 !~ "^[#.]") {print $$1}}' | sort | egrep -v -e '^[^[:alnum:]]' -e '^$@$$'
 
+clean:
+	docker-compose -f ../../docker-compose.yml -f ../../docker-compose.feefeed.yml down && docker volume prune --force
```

Comment:
Wdyt if we move the config to a constant so we can define the intention, i.e. starting full set of services? E.g.

```shell
COMPOSE_FULL_SERVICES=-f ../../docker-compose.yml -f ../../docker-compose.feefeed.yml

docker-compose $(COMPOSE_FULL_SERVICES) down && docker volume prune --force
```